### PR TITLE
Features/improve code doc

### PR DESCRIPTION
**Pull request by merlos**
Achieve a 100% of coverage on the documentation of the iOS App.
(Note that it does not solve #73)

### CODE REVIEW

File: OpenGpxTracker/PreferencesTableViewController.swift [hunk 1]
```diff
@@ -29,21 +29,32 @@ let kDefaultsKeyTileServerInt: String = "TileServerInt"
...
     let defaults = UserDefaults.standard
-    weak var delegate: PreferencesTableViewControllerDelegate?
     
+    /// Delegate for this table view controller.
+    weak var delegate: PreferencesTableViewControllerDelegate?
     
+    /// Does the following:
+    /// 1. Defines the areas for navBar and the Table view
+    /// 2. Sets the title
+    /// 3. Loads the Preferences from defaults (us
```
Comment (vincentneo) [inline OpenGpxTracker/PreferencesTableViewController.swift]: `(us` comment seems incomplete.

File: OpenGpxTracker/ViewController.swift [hunk 2]
```diff
@@ -188,39 +222,78 @@ class ViewController: UIViewController, UIGestureRecognizerDelegate  {
...
+    
+    /// Displays current elapsed time (00:00
     var timeLabel: UILabel
+    
+    /// Label that displays last known speed (in km/h)
     var speedLabel: UILabel
+    
+    /// Distance of the total segments tracked
+    ///
+    /// Displayed in m or km based on the
```
Comment (vincentneo) [inline OpenGpxTracker/ViewController.swift]: incomplete `/// Displayed in m or km based on the`

Same as for var `currentSegmentDistanceLabel`

Comment (merlos) [reply OpenGpxTracker/ViewController.swift]: fixed in 2511fdfc4f8b59f2a1d462f22d5355a64b67ae9e

File: OpenGpxTracker/PreferencesTableViewController.swift (hunk 1 above)
Comment (merlos) [reply OpenGpxTracker/PreferencesTableViewController.swift]: Fixed in 2511fdfc4f8b59f2a1d462f22d5355a64b67ae9e

**Comment by merlos** [comment]
Thanks @vincentneo for the reviews and fixes.

**Pull request merged**